### ISSUE TITLE
Fix E2E tests with leveldb

### ISSUE THREAD

**Issue by themue** [status-im/status-go#873]
# Problem

E2E tests irregularly fail with *leveldb error* locally as well as on Travis. Possibly race condition?

```
$ ./build/bin/statusd -datadir=.ethereumtest/Rinkeby -les -networkid=4 -sync-and-exit=20 -log=WARN -standalone=false -discovery=false
ERROR[04-27|11:03:11] Failed to read database write delay statistic database=/home/travis/gopath/src/github.com/status-im/status-go/.ethereumtest/Rinkeby/StatusIM/lightchaindata err="leveldb: not found"
```

Especially on Travis costs time because blocking of merging.

# Acceptance Criteria

- [ ] Error fixed locally
- [ ] Error fixed on Travis

**Comment by dshulyak** [comment]
This log is not causing *real* errors. It is printed in a separate goroutine which collects stats from ethdb. But for sure we need to understand why it fails at all.

**Comment by themue** [comment]
Thanks for the hint. So it's something not beautiful but it doesn't break the tests. Good to know.

Mind you take a look at https://travis-ci.org/status-im/status-go/jobs/371071051 ? Here I had it quite often.

<blockquote><div><strong><a href="https://travis-ci.org/status-im/status-go/jobs/371071051">Travis CI - Test and Deploy Your Code with Confidence</a></strong></div></blockquote>

**Comment by themue** [comment]
`status-go` is vendoring LevelDB `b89cc31ef7977104127d34c1bd31ebd1a9db2199` which is retrieving a not maintained property `leveldb.writedelay` which is available with `34011bf325bce385408353a30b101fe5e923eb6e`. 

Geth 1.8.5 is vendoring `169b1b37be738edb2813dab48c97a549bcf99bb5` which contains the fix.
